packages/ZbayLite/src/renderer/components/widgets/channels/ChannelInput/ChannelInput.tsx

```diff
@@ -424,6 +424,11 @@ export const ChannelInputComponent: React.FC<ChannelInputProps> = ({
                   html={sanitizedHtml}
                   onChange={onChangeCb}
                   onKeyDown={onKeyDownCb}
+                  onPaste={(e) => {
+                    e.preventDefault()
+                    var text = e.clipboardData.getData('text/plain')
+                    setHtmlMessage(text)
```

Comment:
Doesnt this replace the whole input? I mean if I already have something typed, will this paste in the place my cursor is or will it replace what I have with clipboard content?

Reply:
thats right, changing it